.github/ISSUE_TEMPLATE/--feature_request.md

```diff
@@ -0,0 +1,33 @@
+---
+name: "Feature request"
+about: Suggest an idea for this project
+title: ''
+labels: new feature
+assignees: ''
+
+---
+
+<!-- Please fill out as much of the template as you can. Delete sections if unneccesary -->
+<!-- 
+Make Sure you've checked out Existing Issues
+-->
+
+## Short Description
+<!-- Describe Your Idea -->
+
+
+## **How This Feature will help you/others**
```

Comment:
There should be consistency in the case you are using; either make it title case or just the first letter of first word upper.